MainForm.cs

```diff
@@ -0,0 +1,258 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Windows.Forms;
+using EnumColors;
+
+namespace Task1_GUI
+{
+    public partial class MainForm : Form
+    {
+        public MainForm()
+        {
+            InitializeComponent();
+        }
+        public bool ReadData(int combLen, string[] comb, List<int> nomOfCards, TextBox textBox, List<string> colors)
```

Comment:
variables or methods with bool type should answer some question, like 'IsDataValid'